src/Bridge/Doctrine/Purger/OrmPurger.php

```diff
@@ -79,9 +79,12 @@ public function create(PurgeMode $mode, PurgerInterface $purger = null): PurgerI
 
     /**
      * @inheritdoc
+     * @throws \Doctrine\DBAL\DBALException
      */
     public function purge()
     {
+        $this->manager->getConnection()->prepare('SET FOREIGN_KEY_CHECKS = 0;')->execute();
```

Comment:
Oh, but actually, this is probably MySQL specific, so I don't think it should be here in case someone tries to use it with another kind of database, maybe setting up cascade delete properly will work instead. I'll have a go at that on our project and report back.